css/stylesheet.css

```
@@ -0,0 +1,80 @@
+@charset "utf-8";
+
+body{
+    margin: 0 auto;
+    min-height: 100%;
+    color: #404040;
+    font-size: 15px;
+}
+a{
+    color: #404040;
+    text-decoration: none;
+}
+.navbar{
+    display: flex;
+    justify-content: flex-end;
+    list-style: none;
+}
+.navbar li{
+    margin-right: 20px;
+}
+.colored{
+    color: #ffffff;
+    background: #ed6d35;
+    width: 22px;
+    height: 22px;
+    line-height: 22px;
+    font-size: 11px;
+    display: block;
+    text-align: center;
+    border-radius: 50%;
+}
+.colored a{
+    color: #fff;
+}
+main{
+    min-height: (100vh-106px);
```

```suggestion
    min-height: calc(100vh-106px);
```
あっこうだわ
calc()調べてみてください！

---

あと、引く分にmainのmarginも足してください！

---

修正しました！